src/io/PipeWriter.zig

```diff
@@ -1185,6 +1228,9 @@ pub fn WindowsStreamingWriter(
             this.current_payload = this.outgoing;
             this.outgoing = temp;
             switch (pipe) {
+                .sync_file => {
+                    @panic("sync_file pipe write should not be reachable");
```

Comment:
```suggestion
                    @panic("sync_file pipe write should not be reachable - writeLatin1 in PipeWriter.zig");
```
I like the helpfulness of both this and https://github.com/oven-sh/bun/pull/9398/files#diff-dde34f421656d5d1df29476c8c358985d1b22bae2c505c70e0f3d922e0af9fe5R949 but looking for consistency in direction or information